src/plugins/console/public/plugin.ts

```diff
@@ -55,14 +54,20 @@ export class ConsoleUIPlugin implements Plugin<any, any> {
         defaultMessage: 'Console',
       }),
       enableRouting: false,
-      async mount({ core: { docLinks } }, { element }) {
+      mount: async ({ core: { docLinks, i18n: i18nDep } }, { element }) => {
         const { boot } = await import('./application');
+        const [{ injectedMetadata }] = await getStartServices();
+        const elasticsearchUrl = injectedMetadata.getInjectedVar(
+          'elasticsearchUrl',
+          'http://localhost:9200'
+        ) as string;
         render(
```

Comment:
You may want to take a look at the [conventions for applications](https://github.com/elastic/kibana/blob/master/src/core/CONVENTIONS.md#applications). The main difference between the convention and what you have here is that the `./application` module should export a single `renderApp` function that encapsulates the entire rendering logic (invoking React).

This should be a minor change, but makes this a bit more consistent across the code base.